bootstrap.d/dev-lang.yml

```diff
@@ -87,8 +87,9 @@ tools:
     source:
       subdir: 'ports'
       git: 'https://github.com/rust-lang/cargo.git'
-      tag: '0.53.0'
-      version: '0.53.0'
+      tag: '0.60.0'
+      version: '0.60.0'
+    revision: 2
```

Comment:
Revisions reset when the version increases, so we do not need to bump the revision here.